packages/atomic/src/components/common/tab-manager/tab-button.tsx

```diff
@@ -9,18 +9,23 @@ export interface TabButtonProps {
 
 export const TabButton: FunctionalComponent<TabButtonProps> = (props) => {
   const activeTabClass = props.isActive
-    ? "relative after:content-[''] after:block after:w-full after:h-1 after:absolute after:-bottom-0.5 after:bg-primary after:rounded"
+    ? "active-tab relative after:content-[''] after:block after:w-full after:h-1 after:absolute after:-bottom-0.5 after:bg-primary after:rounded"
     : '';
   const activeTabTextClass = props.isActive ? '' : 'text-neutral-dark';
   return (
-    <div part="button-container" class={activeTabClass}>
+    <li
+      aria-current={props.isActive ? 'true' : false}
```

Comment:
This was broken and will be required for the active tab selector.
```suggestion
      aria-current={props.isActive ? 'true' : 'false'}
```